docs/swagger.yaml

```diff
@@ -0,0 +1,382 @@
+basePath: /content
```

Comment:
The docs should not have been generated this way since we are manually defining docs on this BB.